src/utils/Utils.ts

```diff
@@ -1,9 +1,28 @@
 import chalk from "chalk";
 
-export function exitOnError(e: Error, title?: string) {
-  console.error(
-    chalk.bold.red(title ?? "Fatal error: "),
-    chalk.bold(e.message ?? e ?? "Unknown error")
-  );
+export function exitOnError(e: Error, title?: string, indent?: number) {
+  const titleStyled = title ? chalk.red(title) : "";
+  const indentStyled = indent ? " ".repeat(indent) : "";
+  console.error(chalk.bold(`${indentStyled}${titleStyled} ${e.message}`));
   process.exit(1);
 }
+
+export function printMessage(message: string, title?: string, indent?: number) {
+  const titleStyled = title ? chalk.green(title) : "";
+  const indentStyled = indent ? " ".repeat(indent) : "";
+  console.log(chalk.bold(`${indentStyled}${titleStyled} ${message}`));
+}
+
+export function printError(
+  message: string,
+  title?: string,
+  isWarning = false,
```

Comment:
Node has `console.warn` for warning messages. Maybe we should break this function into `printError` and `printWarning` to incorporate it.

It would also make the function coherent since it's currently called `printError`, but it can print warning messages.

Reply:
I debated if that should be one function or not. They share a lot of code though. Maybe a slight modification here.

Reply:
I made the function switch between error and warn for now.